src/controllers/TorusController.ts

```diff
@@ -631,6 +632,44 @@ export default class TorusController extends BaseController<TorusControllerConfi
     }
   }
 
+  // burn NFT
+  async burnToken(NFTtoBurn: string, senderAddress: string) {
+    try {
```

Comment:
If possible try create a function for generate the burn and close instruction in solanaHelper file
This way we can have tests by simulate the instruction generated